BitwardenShared/Core/Vault/Repositories/VaultRepositoryTests.swift

```diff
@@ -2824,6 +2877,72 @@ class VaultRepositoryTests: BitwardenTestCase { // swiftlint:disable:this type_b
         )
     }
 
+    /// `vaultListPublisher(group:filter:)` returns a publisher for the vault list items filtering cards from
+    /// individual and organizations vaults if restrict item type policy is enabled.
+    @MainActor
+    func test_vaultListPublisher_groups_card_restrictItemTypePolicy() async throws {
+        let cipherOrg1 = Cipher.fixture(id: "1", organizationId: "org1", type: .card)
+        let cipherOrg2 = Cipher.fixture(id: "2", organizationId: "org2", type: .card)
+        let cipher = Cipher.fixture(id: "3", type: .card)
+        cipherService.ciphersSubject.send([cipher, cipherOrg1, cipherOrg2])
+        configService.featureFlagsBool[.removeCardPolicy] = true
+        policyService.policyAppliesToUserPolicies = [
+            .fixture(
+                enabled: true,
+                id: "restrict_item_type",
+                organizationId: "org1",
+                type: .restrictItemTypes,
+            ),
+        ]
+        var iterator = try await subject.vaultListPublisher(
+            group: .card,
+            filter: VaultListFilter(filterType: .allVaults)
+        ).makeAsyncIterator()
+        let vaultListSections = try await iterator.next()
+
+        XCTAssertEqual(
+            vaultListSections,
+            [
+                VaultListSection(
+                    id: "Items",
+                    items: [.fixture(cipherListView: .init(cipher: cipherOrg2))],
+                    name: Localizations.items
+                ),
+            ]
+        )
+    }
+
+    /// `vaultListPublisher(group:filter:)` returns a publisher for the vault list items not filtering cards from
+    /// individual and organizations vaults if restrict item type policy is enabled not enabled.
```

Comment:
⛏️ I believe you just meant `not enabled`:
```suggestion
    /// `vaultListPublisher(group:filter:)` returns a publisher for the vault list items not filtering cards from
    /// individual and organizations vaults if restrict item type policy is not enabled.
```